project4/3700crawler

```diff
@@ -39,26 +39,37 @@ def get_host_path(url):
         path += '?' + url.query
     return (host, path)
 
+def get_status(page):
```

Comment:
'get_status()' and 'get_location()' won't be necessary anymore now that we have `parse_response()`. You can just call `page['Status Code']` or `page['Location']` instead